packages/next-on-pages/docs/supported.md

```diff
@@ -28,12 +26,26 @@ The [Cloudflare workers runtime supports certain Node.js APIs](https://developer
 - `util`
 - `async_hooks`
 
-### External Packages
+## External Packages
 
 You can use any external npm package in your Next.js application, but keep in mind that many packages do rely on Node.js APIs, which, as presented in the previous section could not be supported.
 
+There are also packages which although not requiring node.js APIs don't work with `@cloudflare/next-on-pages` (and/or with Cloudflare's runtime in general) for example those that dynamically evaluate javascript code ([not allowed in the Workers runtime](https://developers.cloudflare.com/workers/runtime-apis/web-standards/#javascript-standards)) or that rely on features not supported by `@cloudflare/next-on-pages`.
+
 If you're application is using a package which relies on unsupported Node.js APIs there is often little to be done, generally the only feasible solutions are to either look for an alternative edge-compatible package or see if the current package could be updated in an edge-compatible manner.
```

Comment:
this paragraph could also be shortened to just: "If you currently use an incompatible package, we suggest that you look for alternatives which often do exist on npm".

Reply:
yes.... but I really wanted to also mention node.js APIs and dynamic evaluation of js code as those are two things that seem to somewhat often trip people over. To keep them around I'll try applying your first suggestion for this if it's ok 🙂 🙏

Reply:
@IgorMinar I've applied your suggestion with some small tweaks

I've also removed the paragraph below as in hindsight it does feel as redundant as the Next.js version one we just removed above

Please have another look 🙂 🙏